src/main/java/domain/result/LottoRank.java

```diff
@@ -0,0 +1,46 @@
+package domain.result;
+
+import domain.budget.Budget;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    NONE_MATCHES(0, Budget.amounts(0), false),
```

Comment:
```suggestion
    NONE_MATCHES(0, 0, false),
```

Budget.amounts 코드가 반복된다면 `LottoRank(int, int, boolean)` 생성자를 통해 중복을 재거할 수 있을 것 같습니다!